Update for 10.0.10

settermjd opened the pull request:
This PR updates the documented system requirements to reflect 10.0.10. It also sorts the operating system requirements list to be easier to quickly scan. 

### Relates To 

https://github.com/owncloud/enterprise/issues/2826.

phil-davis commented on admin_manual/installation/system_requirements.rst:
```diff
@@ -27,22 +28,21 @@ Server
 | Web server       | - Apache 2.4 with ``prefork`` :ref:`apache-mpm-label` and ``mod_php`` |
 +------------------+-----------------------------------------------------------------------+
-| PHP Runtime      | - 5.6+, 7.0, & 7.1                                                    |
+| PHP Runtime      | - 7.1+                                                                |
```
@settermjd have we finally got to the point where we do not support PHP 5.6 (or 7.0)?
They have security fixes for the next few months still, and I have no idea how many people are on those PHP  versions until the last moment.

patrickjahns replied:
See comment below on the commit 🙈  - we are still support php5.6 - it is however recommended to go to 7.1+

settermjd replied:
It's likely that I changed this by mistake, as I referred to composer.json in master, which contains `"php": ">=7.1",`.

settermjd replied:
@patrickjahns, regarding [your other comment](https://github.com/owncloud/documentation/pull/4414#commitcomment-30451429), I'll revert this change.

settermjd commented:
@patrickjahns @phil-davis I've reverted the previous change to the PHP runtime documentation, and added an admonition, encouraging users to migrate to PHP 7.1(+).

settermjd commented:
Sorry, @phil-davis, I misread the documentation (or didn't read it thoroughly enough).

PVince81 commented:
@settermjd no backport means it won't be in the 10.0.10 manual ?

patrickjahns commented:
It's missing 7.2 👎

settermjd commented:
We can backport. I'll add the label.

settermjd commented:
@patrickjahns, give me a sec and I'll add it.

phil-davis commented:
See #4416 for 7.2 support changes.
